opentelemetry/proto/profiles/v1development/profiles.proto

```diff
@@ -192,26 +215,9 @@ message Profile {
   repeated ValueType sample_type = 1;
   // The set of samples recorded in this profile.
   repeated Sample sample = 2;
-  // Mapping from address ranges to the image/binary/library mapped
-  // into that address range.  mapping[0] will be the main binary.
-  // If multiple binaries contribute to the Profile and no main
-  // binary can be identified, mapping[0] has no special meaning.
-  repeated Mapping mapping_table = 3;
-  // Locations referenced by samples via location_indices.
-  repeated Location location_table = 4;
+
   // Array of locations referenced by samples.
-  repeated int32 location_indices = 5;
-  // Functions referenced by locations.
-  repeated Function function_table = 6;
-  // Lookup table for attributes.
-  repeated opentelemetry.proto.common.v1.KeyValue attribute_table = 7;
-  // Represents a mapping between Attribute Keys and Units.
-  repeated AttributeUnit attribute_units = 8;
-  // Lookup table for links.
-  repeated Link link_table = 9;
-  // A common table for strings referenced by various messages.
-  // string_table[0] must always be "".
-  repeated string string_table = 10;
+  repeated int32 location_indices = 3;
```

Comment:
The numbering of the following fields should be adjusted.

Reply:
Number was adjusted with https://github.com/open-telemetry/opentelemetry-proto/pull/644/commits/bfbe49bb4a11e1e58cfb2b32fdf63f3d651f5f52.